packages/v4/patternfly-docs/content/training/html-css/css-variables-and-overrides.md

```diff
@@ -0,0 +1,199 @@
+
+Assign the custom property name `(--pf-c-alert--m-success__title--Color)` that is already inside the `.pf-c-alert{}` block to the global danger color.
+It should look like this: `--pf-c-alert--m-success__title--Color: var(--pf-global--danger-color--100);`
+
+The resulting alert should match Figure 1.
+
+**Figure 1**
+<img src="../img/alert-red-title.png" alt="Alert component with a red title" width="868" />
+
+## Part 2: Overriding PatternFly global properties
+In PatternFly, global properties follow this general formula:
+`--pf-global--concept--PropertyCamelCase--modifier--state`
+
+Global properties are prefixed with the word global.
+ - A `concept` is something like a spacer or main-title.
+ - A `PropertyCamelCase` is something like `BackgroundColor` or `FontSize`.
+ - A `modifier` is something like sm, or lg.
+ - A `state` is something like hover, or expanded. They are concepts, never tied to an element or component. This is incorrect: `--pf-global--h1--FontSize`. This is correct: `--pf-global--FontSize--3xl`.
+
+To explore this concept, override the global link color by setting it to the custom property for danger using <a href="https://codesandbox.io/s/override-global-css-variables-start-llcub8" target="_blank">this codesandbox for part 2</a>. The `index.html` file contains a button and expandable component to demonstrate how changing a global variable has an impact across components.
+
+### Step 1. Familiarize with `index.html`
+Note the two button component and the expandable section component in the `<body>` of the `index.html` file.
+
+### Step 2. Override a global custom property
+
+#### Step 2.1 
+Write the prefix for the global custom property in the `:root` block in the `styles.css` file. Since global custom properties are prefixed with the word global, use that for the beginning of the custom property.
+It should look like this: `--pf-global`
+
+#### Step 2.2 
+Add the concept to the global custom property. The concept for the link color is link.
+The global custom property should now look like this: `--pf-global--link`
+
+#### Step 2.3 
+Add the property to the global custom property. As the color is being modified, add the color property as the next step.
+The global custom property should now look like: `--pf-global--link--Color`
+
+#### Step 2.4 
+Define the value of the global variable. PatternFly's global danger color is: `--pf-global--danger-color--100`.
+Assign the custom property name `(--pf-global--link--Color)` that is already inside of the `:root` block to the global danger color.
+
+It should look like this:
+`--pf-global--link--Color: var(--pf-global--danger-color--100);`
+
+The resulting UI in the codesandbox should match Figure 2. **Note:** The button component and expandable component should inherit different colors.
+
+**Figure 2.**
+<img src="../img/global-link-color-red.png" alt="An inline button and a 'show more' expandable section toggle both colored red" width="200" />
```

Comment:
The previous line says they should inherit different colors, but this shows they are both red? (and they are when I do it)